### PR TITLE
docs: go sdk

### PR DESCRIPTION
**Pull request by qsliu2017**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

go SDK doc

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by aceforeverd** [comment]
[ospp - Golang SDK](https://trello.com/c/7aquDamb/118-ospp-golang-sdk)

File: docs/en/quickstart/go_sdk.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Go SDK Quickstart
+
```
Comment (aceforeverd) [inline docs/en/quickstart/go_sdk.md:2]: specify the requirements: minimal openmdlb version, required components. 

Which ones of go database methods are supported, which is not.

File: docs/en/quickstart/go_sdk.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Go SDK Quickstart
```
Comment (aceforeverd) [inline docs/en/quickstart/go_sdk.md:1]: add the same doc to `docs/zh`

File: docs/en/quickstart/go_sdk.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+Go SDK connects to API server.
+
+```go
+db, err := sql.Open("openmldb", "openmldb://127.0.0.1:8080/test_db")
+```
+
+The DSN schema is
+
+```
+openmldb://HOST[:PORT]/DB_NAME
```
Comment (aceforeverd) [inline docs/en/quickstart/go_sdk.md]: note the `HOST` and `PORT` refer to the api server

**Review by aceforeverd — CHANGES_REQUESTED** [review]

File: docs/en/quickstart/go_sdk.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    panic(err)
+  }
+  // process row ...
+}
+```
+
+### 3. An Example
+
+```go
+package demo
```
Comment (aceforeverd) [inline docs/en/quickstart/go_sdk.md]: package main ?

**Review by aceforeverd — APPROVED** [review]

**Comment by lumianph** [comment]
we do not merge this doc until the v0.7 release, to make as a feature for v0.7

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**